alertmanager/metrics.go

```diff
@@ -108,8 +108,8 @@ func (c *unseeCollector) Collect(ch chan<- prometheus.Metric) {
 					prometheus.GaugeValue,
 					count,
 					am.Name,
-					reciver,
 					state,
+					reciver,
```

Comment:
There's a typo in the name, it should be "receiver", but it's probably not too important?